user_tools/src/spark_rapids_pytools/rapids/tool_ctxt.py

```diff
@@ -121,12 +130,28 @@ def set_local_workdir(self, parent: str):
         self.set_local('depFolder', dep_folder)
         self.logger.info('Dependencies are generated locally in local disk as: %s', dep_folder)
         self.logger.info('Local output folder is set as: %s', exec_root_dir)
-        offline_dir = Utils.resource_path('offline')
-        # Toggle offline mode, if there are any files in offline_dir
-        if os.path.exists(offline_dir) and os.listdir(offline_dir):
-            self.logger.info(Utils.gen_str_header('Offline Mode Enabled', ruler='_', line_width=50))
-            self.logger.info('Offline folder is set as: %s', offline_dir)
-            FSUtil.copy_resource(offline_dir, self.get_cache_folder())
+
+    def load_prepackaged_resources(self):
+        """
+        Checks if the packaging includes the CSP dependencies. If so, it moves the dependencies
+        into the tmp folder. This allows the tool to pick the resources from cache folder.
+        """
+        if not self.are_resources_prepackaged():
+            return
+        self.set_ctxt('offlineModeEnabled', True)
+        self.logger.info(Utils.gen_str_header('Fat Mode Is Enabled',
```

Comment:
Is offlineMode and Fat mode used interchangeably? Or for implementation we use it as offlineMode but for user's it is Fat mode?

Reply:
> Is offlineMode and Fat mode used interchangeably? Or for implementation we use it as offlineMode but for user's it is Fat mode?

- Intenally in the user-tools code (except  prepackageManager), it is used as `offlineMode` which means no web-access
- The documentation/script uses `fat` which is what's visible to the user. Technically we cannot name it `offline` because it still needs to connect to CSP's workspace.